src/lib.rs

```diff
@@ -1101,11 +1098,8 @@ impl Builder {
 }
 
 /// Configuration options for generated bindings.
-///
-/// Deprecated: use a `Builder` instead.
 #[derive(Debug)]
-#[deprecated]
-pub struct BindgenOptions {
+pub(crate) struct BindgenOptions {
```

Comment:
It's in the root, so it can just be private.

Reply:
Makes sense, thanks!